app/spec/controllers/cbv/summaries_controller_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Cbv::SummariesController do
   include PinwheelApiHelper
+  app_date = Date.parse('2024-06-18')
   cbv_flow_props =
```

Comment:
Proper rspec way to set variables is:

```suggestion
  let(:cbv_flow_props) do
    { ... }
  end
```

Reply:
Updated, thanks!